opentelemetry-proto/src/proto/tonic/opentelemetry.proto.trace.v1.rs

```diff
@@ -196,12 +196,12 @@ pub struct Span {
     pub end_time_unix_nano: u64,
     /// attributes is a collection of key/value pairs. Note, global attributes
     /// like server name can be set using the resource API. Examples of attributes:
-    ///
+    /// ```ignore
```

Comment:
this is auto-generated code right? we should remove this file from the check.

Reply:
yeah, you are right. didn't realize they were proto based.